Makefile

```diff
@@ -26,7 +33,14 @@ check-examples:
 			echo "Checking example: $${example}"; \
 			VALUES=$$(find $${EXAMPLES_DIR}/$${example} -name *values.yaml); \
 			for value in $${VALUES}; do \
+				helm dependency build charts/$${chart_name}; \
 				helm template example charts/$${chart_name} --values $${value} --output-dir "${TMP_DIRECTORY}/$${example}"; \
+				SUBCHARTS_DIR=${TMP_DIRECTORY}/$${example}/$${chart_name}/charts; \
+				SUBCHARTS=$$(find $${SUBCHARTS_DIR} -type d -maxdepth 1 -mindepth 1 -exec basename \{\} \;); \
+				for subchart in $${SUBCHARTS}; do \
+					mkdir -p "${TMP_DIRECTORY}/$${example}/$${chart_name}/templates/$${subchart}"; \
+					mv ${TMP_DIRECTORY}/$${example}/$${chart_name}}/charts/$${subchart}/templates/* "${TMP_DIRECTORY}/$${example}/$${chart_name}/templates/$${subchart}"; \
+				done; \
 			done; \
 			if diff "$${EXAMPLES_DIR}/$${example}/rendered" "${TMP_DIRECTORY}/$${example}/$${chart_name}/templates" > /dev/null; then \
```

Comment:
I don't think this diff is including the subchart.

Reply:
@povilasv I think adding a `-r` flag to the diff command will do the trick.  The subchart files are inside `/templates`, but the diff command isn't currently recursively diffing.